module5/src/main/java/bean/EmptySubjectException.java

```diff
@@ -0,0 +1,20 @@
+package bean;
+
+public class EmptySubjectException extends MyException {
+    private final String studentName;
```

Comment:
No need to use this field.
Just create constructor:
```
    public EmptySubjectException(String message) {
        super(message);
    }
```

Here and in other custom messages.